zenodo/config.py

```diff
@@ -867,3 +867,21 @@ def _(x):
 # Proxy configuration
 #: Number of proxies in front of application.
 WSGI_PROXIES = 0
+
+
+# Indexer
+# =======
+#: Provide a custom record_to_index function for invenio-indexer
+INDEXER_RECORD_TO_INDEX = "zenodo.modules.indexer.utils.record_to_index"
+INDEXER_SCHEMA_TO_INDEX_MAP = {
+    'http://zenodo.org/schemas/records/record-v1.0.0.json': ('records-record-v1.0.0', 'record-v1.0.0'),
```

Comment:
Can't we reduce this to:

```python
{
   '<index name>': '<mapping name>',
  #...
}
```
and use schema_to_index, to provide us with the index name.....?